src/components/Footer/Footer.module.css

```
@@ -0,0 +1,16 @@
+.container {
+  background-color: var(--nomad_black);
+  width: 100%;
+  height: 16rem;
+  position: absolute;
+  bottom: 0;
+  left: 0;
```

컴포넌트 하단에 위치해서 position: absolute가 필요없을 것 같아요.
```suggestion
display: flex;
justify-content: center;

```